src/BuildLoop.cpp

```diff
@@ -74,45 +75,24 @@ static void LogFirstTimeEnqueue(MemAllocLinear* scratch, RuntimeNode* enqueuedNo
     LogStructured(&msg);
 }
 
-static int EnqueueNodeListReversedWithoutWakingAwaitersOnlyWhenNotEnqueuedBefore(BuildQueue* queue, MemAllocLinear* scratch, const FrozenArray<int32_t>& nodesToEnqueue, RuntimeNode* enqueingNode)
+static int EnqueueNodeListWithoutWakingAwaiters(BuildQueue* queue, MemAllocLinear* scratch, const FrozenArray<int32_t>& nodesToEnqueue, RuntimeNode* enqueingNode)
 {
-    int enqueue_count = 0;
+    int placed_on_workstack_count = 0;
 
     //we enqueue dependency lists in reverse. because our semantics say that the most urgent dependencies are in the list first, it's important that they
     //end up on the top of the workstack, so they'll be processed first.
```

Comment:
Does this comment still make sense here? It looks like we process nodesToEnqueue in "forward" order now.

Reply:
fixed 8882e09